packages/gatsby-theme-docs/bin/create-release-note

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env node
```

Comment:
Better to name this file `create-release-note.sh` (with the `.sh` extension).

Reply:
Really?  To me, `.sh` is for bash script and not JS.  Editors would pick up the wrong syntax highlighting, too.  I would rather consider to keep it "js" and  define an extensionless binary name in package.json that will then be the actual command.

Reply:
Sorry you're right, I meant to say to add the file extension (which is obviously `.js` in this case) 😅

Reply:
Ahh, okay thanks 😄